lms/envs/common.py

```diff
@@ -212,9 +212,9 @@
     # when enrollment exceeds this number
     'MAX_ENROLLMENT_INSTR_BUTTONS': 200,
 
-    # Grade calculation started from the new instructor dashboard will write
-    # grades CSV files to S3 and give links for downloads.
-    'ENABLE_S3_GRADE_DOWNLOADS': False,
+    # Grade calculation started from the instructor dashboard will write grades
+    # CSV files to the configured storage backend and give links for downloads.
+    'ENABLE_GRADE_DOWNLOADS': False,
```

Comment:
This change is backward-compatible: https://github.com/edx/edx-platform/pull/11286/files#diff-8a23909bc8d201fba8525b1dd6396247R181